use_cases/eluc/persistence/persistors/hf_persistor.py

```diff
@@ -0,0 +1,69 @@
+"""
+Persistor for models to and from HuggingFace repo.
+"""
+from pathlib import Path
+
+from huggingface_hub import HfApi, snapshot_download
+
+from persistence.serializers.serializer import Serializer
+
+class HuggingFacePersistor():
+    """
+    Persists models to and from HuggingFace repo.
+    """
+    def __init__(self, file_serializer: Serializer):
+        self.file_serializer = file_serializer
+
+    def write_readme(self, model_path: str):
+        """
+        Writes readme to model save path to upload.
+        TODO: Need to add more info to the readme and make it a proper template.
+        """
+        model_path = Path(model_path)
+        with open(model_path / "README.md", "w", encoding="utf-8") as file:
+            file.write("This is a demo model created for project resilience")
```

Comment:
Some boilerplate huggingface code that probably will need to be fleshed out more in the future